src/modules/token/fungible/utils/lsk.js

```diff
@@ -10,9 +10,22 @@ BigNumber.config({ ERRORS: false });
  * @param {Strong|Number} value - Value in Beddow
  * @returns {BigNumber} Value converted to LSK
  */
-export const fromRawLsk = value => (
-  new BigNumber(value || 0).dividedBy(new BigNumber(10).pow(8)).toFixed()
-);
+export const fromRawLsk = (value) =>
+  new BigNumber(value || 0).dividedBy(new BigNumber(10).pow(8)).toFixed();
+
+/**
+ * Convert Beddow to LSK
+ *
+ * @param {Strong|Number} amount - Amount value to be converted
+ * @param {Strong|Number} token - Token value merged with its equivalent metadata
+ * @returns {BigNumber} Amount value converted to the token symbol's denom
```

Comment:
A few things, Number is the documentation defines the interface, and number defines an instance of the Number interface. I think you need the later. 
The name is confusing. Since you are converting to a different denomination of the same token. I think `toOtherDenom`, or `changeDemnom`  could be a name.
The documentation doesn't clarify what should be inside the token. The default value is an empty object, while the documentation suggests a Number.

Reply:
Also, we already have two other methods. toRawLsk and fromRawLsk. Either refactor the components that use them or event better, create a token to do so.

Reply:
So we already plan of having a ticket that would integrate all other usage of `toRawLsk` and `fromRawLsk` with the new util function. Though I think having a naming called `changeDemnom` is better